frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx

```diff
@@ -120,7 +121,35 @@ export const DataPermissionsHelp = () => {
                 icon="permissions_limited"
                 iconColor="brand"
                 name={t`Sandboxed (Pro)`}
-                description={t`Let's you specify row and column-level permissions. Can be set up via user attributes and SSO.`}
+                description={t`Lets you specify row and column-level permissions. Can be set up via user attributes and SSO.`}
```

Comment:
Technically needs another hyphen:

```suggestion
                description={t`Lets you specify row- and column-level permissions. Can be set up via user attributes and SSO.`}
```

I slightly prefer "via user attributes or SSO"

Reply:
I think that's even better!